lib/database/migrations/20231011103554-create-lhc-period.js

```diff
@@ -106,28 +36,43 @@ module.exports = {
             allowNull: true,
         }, { transaction });
 
-        const updateStmt = await prepareRunsUpdateStatement(
-            await createAndGetPeriods({ queryInterface, transaction }),
-            { queryInterface, transaction },
-        );
-        if (updateStmt) {
-            await queryInterface.sequelize.query(updateStmt, { transaction });
-        }
+        const insertPeriodsStmt = `
+            INSERT INTO lhc_periods(name)
+                SELECT DISTINCT lhc_period 
+                FROM runs
+                    WHERE lhc_period IS NOT NULL
+            ON DUPLICATE KEY UPDATE name = values(name)
```

Comment:
You have to create the table first. I don't think your `ON DUPPLICATE KEY UPDATE` is a good idea. First the table has been created right before so you must not have dupplicate, and I think that you should almost NEVER update an ID